app/db/crud/document.py

```diff
@@ -108,7 +108,7 @@ def get_family_document_and_context(
         content_type=physical_document.content_type,
         language=(visible_languages[0] if visible_languages else ""),
         languages=visible_languages,
-        document_type=document.document_type,
+        document_type=document.valid_metadata["type"][0],
```

Comment:
Can I ask we validate the data after line 81 to fail gracefully ... otherwise if we get bad data in the database we are going to fall flat :)